### PR TITLE
move any into an array, add time format

### PR DESCRIPTION
**Pull request by fearthecowboy**

### CODE REVIEW

File: codemodel/.resources/all-in-one/json/code-model.json [hunk 1]
```diff
@@ -1710,6 +1731,13 @@
             "$ref": "#/definitions/DateSchema"
           }
         },
+        "times": {
+          "description": "a time",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApiVersionimeSchema"
```
Comment (daviwil) [inline codemodel/.resources/all-in-one/json/code-model.json]: Something weird happened with the definition name here: `ApiVerisonimeSchema`

**Review by daviwil — APPROVED** [review]
The schema name of the TimeSchema is a little screwed up somehow but otherwise LGTM

File: codemodel/.resources/all-in-one/json/code-model.json (hunk 1 above)
Comment (fearthecowboy) [reply codemodel/.resources/all-in-one/json/code-model.json]: O_O

File: codemodel/.scripts/generate-schema.js [hunk 2]
```diff
@@ -128,7 +128,7 @@ function fixmodel(schema) {
     .replace(/T_3/g, 'Extensions')
     .replace(/Protocols<Protocol>/g, 'Protocols')
     .replace(/Languages<Language>/g, 'Languages')
-    .replace(/definitions\/T/g, 'definitions/ApiVersion')
+    .replace(/definitions\/T"/g, 'definitions/ApiVersion"')
```
Comment (daviwil) [inline codemodel/.scripts/generate-schema.js:131]: Oof!

**Review by daviwil — APPROVED** [review]

**Pull request merged**